main.py

```diff
@@ -446,11 +446,15 @@ async def elm_command(ctx: SlashContext, elm_type: str, elm_id: str, extras: str
 
 def get_elm(elm_type: str, elm_id: str | int, suffix: str = "") -> dict:
     res = requests.get(config["api_url"] + f"api/0.6/{elm_type}/{elm_id}.json" + suffix)
+    code = res.status_code
+    if code == 410:
+        raise ValueError(f"`{elm_type.capitalize()}` `{elm_id}` has been deleted.")
+    elif code == 404:
+        raise ValueError(f"`{elm_type.capitalize()}` `{elm_id}` has never existed.")
```

Comment:
It would probably be better to not have the element name in a ` ` bit.
```suggestion
    if code == 410:
        raise ValueError(f"{elm_type.capitalize()} `{elm_id}` has been deleted.")
    elif code == 404:
        raise ValueError(f"{elm_type.capitalize()} `{elm_id}` has never existed.")
```